### DISCUSSION TITLE
Project is DEAD....?

### DISCUSSION THREAD

**Discussion by Nicalion** [Melledy/Grasscutter#2503] [Support]
Is it dead or why no upades since 4.0.0?

**Comment by JinoSUS** [comment]
100%

**Comment by marcussacana** [comment]
There are no updates because mihoyo put an stronger protection the required parts of the game that grasscutter works, unless someone with vmprotect expetize feel like in helping the project I doubt there will be a update, or maybe find someone and contract to do it.
